<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1461</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="numbersEnterEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>270</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>270</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1001</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="arrayVisualization">
     <property name="bottomMargin">
      <number>6</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>320</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeArray">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>320</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>85</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>440</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="sortStartHeap">
       <property name="text">
        <string>Heap Sorting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortStarting">
       <property name="text">
        <string>Quick sorting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortStartmerge">
       <property name="text">
        <string>Merge sorting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortStaringinterpolation">
       <property name="text">
        <string>Interpolation sorting</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="searchValue">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>590</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searching">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>590</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QLabel" name="binPowResult">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>630</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1461</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
